<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>711</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>721</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton{
background-color: rgba(85,98,112,255);
color: rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#pushButton:pressed{
padding-left: 5px;
padding-top:5px;
background-color: rgba(255,107,107,255);
background-position:calc(100%-10px)center;
}
QPushButton#pushButton:hover{
background-color:rgba(255,107,107,255);
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>381</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for QLabel with a solid lightest blue background */
QLabel {
    background-color: #FFFFFF; /* Lightest blue background color */
    border-radius: 10px; /* Rounded corners */
    padding: 10px; /* Padding around the content */
    color: #333333; /* Text color */
    font-family: Arial, sans-serif; /* Font family */
    font-size: 16px; /* Font size */
    text-align: center; /* Center-align text */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>25</y>
      <width>341</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for QLabel with a darker gradient background */
QLabel {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 0, 128, 255), stop:1 rgba(0, 0, 128, 255));
    border-radius: 10px; /* Rounded corners */
    padding: 10px; /* Padding around the content */
    color: #FFFFFF; /* Text color (white) */
    font-family: Arial, sans-serif; /* Font family */
    font-size: 16px; /* Font size */
    text-align: center; /* Center-align text */
}

/* QLabel when hovered */
QLabel:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 128, 255), stop:1 rgba(85, 0, 128, 255));
}

/* QLabel when clicked (pressed) */
QLabel:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(128, 0, 0, 255), stop:1 rgba(0, 0, 85, 255));
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Javanese Text</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0,0,0,200);
</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46, 82, 101, 200);
color:rgba(0,0,0);
padding-bottom:7px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>UserName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46, 82, 101, 200);
color:rgba(0,0,0);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>PassWord</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for QPushButton with solid blue background and white text */
QPushButton {
    background-color: #0000A5; /* Solid blue background color */
    color: #FFFFFF; /* Text color (white) */
    border: 1px solid #333333; /* Border style */
    border-radius: 5px; /* Rounded corners */
    padding: 10px 20px; /* Padding around the content */
    font-family: Arial, sans-serif; /* Font family */
    font-size: 18px; /* Font size */
}

/* QPushButton when hovered */
QPushButton:hover {
    background-color: #0b00d7; /* Slightly lighter blue on hover */
	color: yellow;
}
</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>34</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:center;
color:rgba(255,255,255,220);
margin-left:center;
margin-right:center;</string>
    </property>
    <property name="text">
     <string>QLibrary</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>5</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>444</width>
      <height>444</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>5</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>D:/Downloaded Files/books (2).png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
